dbt/seeds/spatial/spatial.stadium.csv

```diff
@@ -0,0 +1,8 @@
+name,lon,lat,x_3435,y_3435,year,date_opened,year
+Soldier Field,-87.6165891,41.862322,1179603,1893122,2022,1924

```

Comment:
year is a required column in the dist_pin_to_geography macro. Having it as 2024 was not resulting in any joins.

Reply:
That's because the macro looks for that year of parcel data, and 2024 doesn't yet exist. You can actually just merge `date_opened` and `year`, keeping the open year for each stadium (in a column called `year`). That should fill appropriately.